components/mediation/inbound-endpoints/org.wso2.micro.integrator.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/generic/GenericProcessor.java

```diff
@@ -51,7 +51,7 @@ public class GenericProcessor extends InboundRequestProcessorImpl implements Tas
 
     public GenericProcessor(String name, String classImpl, Properties properties, long scanInterval,
```

Comment:
Better to overload the constructor.

Reply:
Resolved by https://github.com/wso2/product-micro-integrator/pull/4320/commits/e5fad412e1e767a604e3caf14c4c86a44b177d20